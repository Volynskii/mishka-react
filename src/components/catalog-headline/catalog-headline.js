import './catalog-headline.scss';
import React from "react";


export const CatalogHeadline = () => {

    return (
        <article className="catalog">
   <h5 className="catalog__headline">Каталог товаров</h5>

<ul className="catalog__list">
    <li className="catalog__list__item">

    <div  className="catalog__list__item__image"/>
        <div className="catalog__list__item__wrapper">
    <section className="catalog__list__item__param">
    <div className="catalog__list__item__param__name">Зайчик-попрыгайчик</div>
    <div className="catalog__list__item__param__description">Рост 30 см, вес 200 г </div>
    </section>
    <ul className="catalog__list__item__order">
        <li className="catalog__list__item__order__price">  1 200 руб.</li>
        <li className="catalog__list__item__order__basket"> </li>
    </ul>
        </div>
</li>

    <li className="catalog__list__item">

        <div  className="catalog__list__item__image"/>
        <div className="catalog__list__item__wrapper">
            <section className="catalog__list__item__param">
                <div className="catalog__list__item__param__name">Зайчик-попрыгайчик</div>
                <div className="catalog__list__item__param__description">Рост 30 см, вес 200 г </div>
            </section>
            <ul className="catalog__list__item__order">
                <li className="catalog__list__item__order__price">  1 200 руб.</li>
                <li className="catalog__list__item__order__basket"> </li>
            </ul>
        </div>
    </li>

    <li className="catalog__list__item">

        <div  className="catalog__list__item__image"/>
        <div className="catalog__list__item__wrapper">
            <section className="catalog__list__item__param">
                <div className="catalog__list__item__param__name">Зайчик-попрыгайчик</div>
                <div className="catalog__list__item__param__description">Рост 30 см, вес 200 г </div>
            </section>
            <ul className="catalog__list__item__order">
                <li className="catalog__list__item__order__price">  1 200 руб.</li>
                <li className="catalog__list__item__order__basket"> </li>
            </ul>
        </div>
    </li>






</ul>
        </article>
    )
};
