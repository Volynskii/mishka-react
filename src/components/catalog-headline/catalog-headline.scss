@import "../../utilities/variables";

/*tablet-tablet-tablet */
@media (min-width: $tablet-width-min) and (max-width: $tablet-width-max) {
  .catalog {
    display: flex;
    flex-direction: column;
    justify-content: flex-start;
    border: 1px solid;
    width: 770px;
    margin: 0;
    margin-left: auto;
    margin-right: auto;
    margin-bottom: 57px;
  }
  .catalog__headline {
    text-align: center;
    font-size: $headline-font-size-tablet;
    line-height: $headline-font-size-tablet;
    font-weight: bold;
    margin: 0;
    margin-bottom: 170px;
  }
  .catalog__list {
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    margin: 0;
    padding: 0;
    width: 670px;
    min-height: 1110px;
    list-style-type: none;
    margin-left: auto;
    margin-right: auto;
  }
  .catalog__list__item {
    display: flex;
    width: 670px;
    height: 334px;
    border: 1px solid;
  }

  .catalog__list__item__image {
    width: 50%;
    height: 334px;
    border: 1px solid;
    background-image: url("./images/photo-1-desktop.png");
    background-repeat: no-repeat;

  }
  .catalog__list__item__wrapper {
    width: 50%;
    display: flex;
    flex-direction: column;
    justify-content: space-between;
  }
  .catalog__list__item__param {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    height: 123px;
    width: 270px;
    border: 1px solid;
    margin-left: auto;
    margin-right: auto;
  }
  .catalog__list__item__param__name {
    font-size: 20px;
    line-height: 24px;
    font-weight: bold;
    margin-bottom: 10px;
  }
  .catalog__list__item__param__description {
    font-size: 15px;
    line-height: 30px;
  }
  .catalog__list__item__order {
    list-style-type: none;
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin: 0;
    margin-left: auto;
    margin-right: auto;
    border: 1px solid;
    width: 256px;
    height: 76px;
  }
  .catalog__list__item__order__price {
    font-size: 17px;
    font-weight: bold;
  }
  .catalog__list__item__order__basket {
    height: 18px;
    width: 22px;
    background-image: url("../../images/header/desktop/cart.png");
  }
  .catalog__list__item__order__basket:hover {
    background-image: url("../../images/header/desktop/cart-hover.png");
  }
  .catalog__list__item__order__basket:active {
    background-image: url("../../images/header/desktop/cart-active.png");
  }
};



/*desktop-desktop-desktop */
@media (min-width: $desktop-width-min) {
  .catalog {
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    border: 1px solid;
    width: 1050px;
    height: 940px;
    margin: 0;
    margin-left: auto;
    margin-right: auto;
    margin-bottom: 57px;
  }
  .catalog__headline {
    text-align: center;
    font-size: $headline-font-size-desktop;
    line-height: $headline-font-size-desktop;
    font-weight: bold;
    margin: 0;
  }
  .catalog__list {
    display: flex;
    justify-content: space-between;
    margin: 0;
    padding: 0;
    width: 1050px;
    list-style-type: none;
  }
  .catalog__list__item {
    width: 316px;
    height: 660px;
    border: 1px solid;
  }
  .catalog__list__item__image {
    height: 460px;
    border: 1px solid;
    background-image: url("./images/photo-1-desktop.png");

  }
  .catalog__list__item__param {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    height: 123px;
    border: 1px solid;
  }
  .catalog__list__item__param__name {
    font-size: 20px;
    line-height: 24px;
    font-weight: bold;
    margin-bottom: 10px;
  }
  .catalog__list__item__param__description {
    font-size: 15px;
    line-height: 30px;
  }
  .catalog__list__item__order {
    list-style-type: none;
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin: 0;
    margin-left: auto;
    margin-right: auto;
    border: 1px solid;
    width: 256px;
    height: 76px;
  }
  .catalog__list__item__order__price {
    font-size: 17px;
    font-weight: bold;
  }
  .catalog__list__item__order__basket {
    height: 18px;
    width: 22px;
    background-image: url("../../images/header/desktop/cart.png");
  }
  .catalog__list__item__order__basket:hover {
    background-image: url("../../images/header/desktop/cart-hover.png");
  }
  .catalog__list__item__order__basket:active {
    background-image: url("../../images/header/desktop/cart-active.png");
  }
  .tabs__item--active {
    background-image: url("../../images/header/desktop/cart-hover.png");
  }
};
