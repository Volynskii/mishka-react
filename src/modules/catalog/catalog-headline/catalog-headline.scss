@import "../../../utilities/variables";


/*mobile-mobile-mobile */
@media (min-width: $mobile-width-min) and (max-width: $mobile-width-max) {
  .catalog {
    display: flex;
    flex-direction: column;
    justify-content: flex-start;
    align-items: center;
    width: 100%;
    margin: 0;
    margin-top: 100px;

  }
  .catalog__headline {
    width: 262px;
    height: 180px;
    background-color: #f1f1f1;
    display: flex;
    justify-content: center;
    align-items: center;
    font-size: $headline-font-size-mobile;
    line-height: $headline-font-size-mobile;
    font-weight: bold;
    margin: 0;
  }
  .catalog__list {
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    align-items: center;
    margin: 0;
    padding: 0;
    width: 87%;
    min-height: 1110px;
    list-style-type: none;

  }
  .catalog__list__item {
    display: flex;
    flex-direction: column;
    align-items: center;
    width: 260px;
    min-height: 375px;
    border: 2px solid #efefef;
    margin-bottom: 30px;
  }

  .catalog__list__item__image {
    width: 100%;
    height: 195px;
    border: 1px solid;
    display: flex;
    justify-content: center;
    align-items: flex-start;
   // background-color: #e5e5e5;
    background-color: #d3d2d0;

  }
  .catalog__list__item__image-picture {
    width: 150px;
    height: 260px;
    background-color: #d3d2d0;
  }
  .catalog__list__item__wrapper {
    width: 210px;
    display: flex;
    flex-direction: column;
    justify-content: space-between;
  }
  .catalog__list__item__param {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: flex-start;
    height: 105px;
    width: 200px;
   // border: 1px solid red;
    margin-left: auto;
    margin-right: auto;
  }
  .catalog__list__item__param__name {
    font-size: 17px;
    line-height: 24px;
    font-weight: bold;
    margin-bottom: 10px;
  }
  .catalog__list__item__param__description {
    width: 200px;
text-align: left;
    font-size: 13px;
    line-height: 30px;
  }
  .catalog__list__item__order {
    list-style-type: none;
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin: 0;
    margin-left: auto;
    margin-right: auto;
    border-top: 3px solid #f0f0f0;
    width: 200px;
    height: 76px;
    padding: 0;
  }
  .catalog__list__item__order__price {
    font-size: 15px;
    font-weight: bold;
  }
  .catalog__list__item__order__basket {
    height: 18px;
    width: 22px;
    background-image: url("../../../images/header/desktop/cart.png");
  }
  .catalog__list__item__order__basket:hover {
    background-image: url("../../../images/header/desktop/cart-hover.png");
  }
  .catalog__list__item__order__basket:active {
    background-image: url("../../../images/header/desktop/cart-active.png");
  }
  .tabs__item--active {
    background-image: url("../../../images/header/desktop/cart-hover.png");
  }
};

/*tablet-tablet-tablet */
@media (min-width: $tablet-width-min) and (max-width: $tablet-width-max) {
  .catalog {
    display: flex;
    flex-direction: column;
    justify-content: flex-start;
    //border: 1px solid;
    width: 100%;
    margin: 0;
    margin-left: auto;
    margin-right: auto;
    margin-bottom: 57px;
    background-image: url("../../../images/header/desktop/mishka-catalog-desktop.png");
    background-size: 770px;
    background-position: top;
    background-repeat: no-repeat;
  }
  .catalog__headline {
    text-align: center;
    font-size: $headline-font-size-tablet;
    line-height: $headline-font-size-tablet;
    font-weight: bold;
    margin: 0;
    margin-bottom: 170px;
  }
  .catalog__list {
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    align-items: center;
    margin: 0;
    padding: 0;
    width: 87%;
    min-height: 1110px;
    list-style-type: none;
    margin-left: auto;
    margin-right: auto;
  }
  .catalog__list__item {
    display: flex;
    width: 87%;
    height: 334px;
    //border: 1px solid;
  }

  .catalog__list__item__image {
    width: 50%;
    height: 334px;
    //border: 1px solid;
    display: flex;
    justify-content: center;
   // background-color: #e5e5e5;

  }
  .catalog__list__item__wrapper {
    border-top: 3px solid #eeeef0;
    border-right: 3px solid #eeeef0;
    border-bottom: 3px solid #eeeef0;
    width: 50%;
    display: flex;
    flex-direction: column;
    justify-content: space-between;
  }
  .catalog__list__item__param {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    height: 123px;
    width: 270px;
    //border: 1px solid;
    margin-left: auto;
    margin-right: auto;
  }
  .catalog__list__item__param__name {
    width: 210px;
    font-size: 20px;
    line-height: 24px;
    font-weight: bold;
    margin-bottom: 10px;
  }
  .catalog__list__item__param__description {
    width: 210px;
    font-size: 15px;
    line-height: 30px;
  }
  .catalog__list__item__order {
    list-style-type: none;
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin: 0;
    margin-left: auto;
    margin-right: auto;
    border-top: 3px solid #eeeef0;
    width: 256px;
    height: 76px;
  }
  .catalog__list__item__order__price {
    font-size: 17px;
    font-weight: bold;
  }
  .catalog__list__item__order__basket {
    height: 18px;
    width: 22px;
    background-image: url("../../../images/header/desktop/cart.png");
  }
  .catalog__list__item__order__basket:hover {
    background-image: url("../../../images/header/desktop/cart-hover.png");
  }
  .catalog__list__item__order__basket:active {
    background-image: url("../../../images/header/desktop/cart-active.png");
  }
  .tabs__item--active {
    background-image: url("../../../images/header/desktop/cart-hover.png");
  }
};



/*desktop-desktop-desktop */
@media (min-width: $desktop-width-min) {
  .catalog {
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    //border: 1px solid;
    width: 1050px;
    height: 880px;
    margin: 0;
    margin-left: auto;
    margin-right: auto;
    margin-bottom: 57px;
    background-image: url("../../../images/header/desktop/mishka-catalog-desktopv2.png");
    background-position: top;
    background-repeat: no-repeat;
  }
  .catalog__headline {
    text-align: center;
    font-size: $headline-font-size-desktop;
    line-height: $headline-font-size-desktop;
    font-weight: bold;
    margin: 0;
    margin-top: 30px;
  }
  .catalog__list {
    display: flex;
    justify-content: space-between;
    margin: 0;
    padding: 0;
    width: 1050px;
    list-style-type: none;
  }
  .catalog__list__item {
    width: 320px;
    height: 660px;
    //border: 1px solid;
  }
  .catalog__list__item__image {
    height: 460px;
    //border: 1px solid;
    background-image: url("./images/photo-1-desktop.png");

  }
  .catalog__list__item__param {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    height: 123px;
    border-left: 2px solid #e5e0dc;
    border-right: 2px solid #e5e0dc;

  }
  .catalog__list__item__param__name {
    width: 260px;
    font-size: 20px;
    line-height: 24px;
    font-weight: bold;
    margin-bottom: 10px;
    text-align: left;
  }
  .catalog__list__item__param__description {
    font-size: 15px;
    line-height: 30px;
  }
  .catalog__list__item__order {
    border-left: 2px solid #e5e0dc;
    border-right: 2px solid #e5e0dc;
    border-bottom: 2px solid #e5e0dc;
    list-style-type: none;
    display: flex;
    justify-content: space-between;
    align-items: center;
    margin: 0;
    margin-left: auto;
    margin-right: auto;
    padding: 0;

    width: 316px;
    height: 76px;
  }
  .catalog__list__item__order__price {
    font-size: 17px;
    font-weight: bold;
    margin-left: 30px;
  }
  .catalog__list__item__order__basket {
    height: 18px;
    width: 22px;
    background-image: url("../../../images/header/desktop/cart.png");
    margin-right: 30px;
  }
  .catalog__list__item__order__basket:hover {
    background-image: url("../../../images/header/desktop/cart-hover.png");
    cursor: pointer;
  }
  .catalog__list__item__order__basket:active {
    background-image: url("../../../images/header/desktop/cart-active.png");
  }
  .tabs__item--active {
    background-image: url("../../../images/header/desktop/cart-hover.png");
  }
};
