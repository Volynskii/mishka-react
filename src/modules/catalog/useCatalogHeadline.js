import {useSelector} from "react-redux";

export const useCatalogHeadline = () => {

    const catalogItems = useSelector((state) => state.basketItems.catalogItems);
    const basketItems = useSelector((state) => state.basketItems.basketItems);

    return {catalogItems,basketItems}
};
