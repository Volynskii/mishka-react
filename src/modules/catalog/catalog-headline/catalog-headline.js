import './catalog-headline.scss';
import React from "react";
import CatalogItem from "./components/catalogItem";

export const CatalogHeadline = ({catalogItemsData,basketItemsData}) => {

    return (
        <article className="catalog">
            <div className="catalog__headline">Каталог товаров
   {/*<h5>Каталог товаров</h5>*/}
            </div>
<ul className="catalog__list">
    {catalogItemsData.map((catalogItem, index) => {
        return (

            <CatalogItem
        key={catalogItemsData[index].id}
        basketItem={basketItemsData}
        catalogItem={catalogItem}
        />
        );
    })}
</ul>
        </article>
    )
};
