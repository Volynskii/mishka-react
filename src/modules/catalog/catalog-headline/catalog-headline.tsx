import './catalog-headline.scss';
import React from "react";
import CatalogItem from "./components/catalogItem";

type catalogContainerProps = {
    catalogItemsData: {
        id: number;
        src: string;
        itemPrice: number;
        itemName: string;
        itemParamOneName: string;
        itemParamOneValue: string;
        itemParamOneUnit : string;
        itemParamTwoValue: string;
        itemParamTwoName: string;
        itemParamTwoUnit: string;
        itemQuantity: number;}[];

    basketItemsData: {
            id: number;
            src: string;
            itemPrice: number;
            itemName: string;
            itemParamOneName: string;
            itemParamOneValue: string;
            itemParamOneUnit : string;
            itemParamTwoValue: string;
            itemParamTwoName: string;
            itemParamTwoUnit: string;
            itemQuantity: number;}[];

}

export const CatalogHeadline: React.FC<catalogContainerProps> = ({catalogItemsData,basketItemsData}) => {

    return (
        <article className="catalog">
            <div className="catalog__headline">Каталог товаров

            </div>
            <ul className="catalog__list">
                {catalogItemsData.map((catalogItem: {
                    id: number;
                    src: string;
                    itemPrice: number;
                    itemName: string;
                    itemParamOneName: string;
                    itemParamOneValue: string;
                    itemParamOneUnit : string;
                    itemParamTwoValue: string;
                    itemParamTwoName: string;
                    itemParamTwoUnit: string;
                    itemQuantity: number;
                },


                ) => {
                    return (

                        <CatalogItem
                            key={catalogItem.id}
                            basketItem={basketItemsData}
                            catalogItem={catalogItem}
                        />
                    );
                })}
            </ul>
        </article>
    )
};
