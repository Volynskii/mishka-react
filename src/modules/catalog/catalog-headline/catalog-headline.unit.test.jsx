import React, {useRef} from "react";
import {mount} from "enzyme";
import {Provider, default as redux} from "react-redux";
import configureStore from "redux-mock-store";
import renderer, {act} from "react-test-renderer";
import catalogItems from "../../../mocks/catalog-items-mock";
import {useCatalogHeadline} from "../useCatalogHeadline";
import {CatalogHeadline} from "./catalog-headline";

const basketItems = [
    {id:0,
        src: "./images/catalog/catalog-item-1.png",
        itemPrice: 1200,
        itemName:'Зайчик-попрыгайчик',
        itemParamOneName:'рост',
        itemParamOneValue: 30,
        itemParamOneUnit: 'см',
        itemParamTwoName: 'вес',
        itemParamTwoValue: 200,
        itemParamTwoUnit: 'г',
        itemQuantity: 1
    },
    {id:1,
        src: "./images/catalog/catalog-item-2.png",
        itemPrice: 690,
        itemName:'Корзинка для белья',
        itemParamOneName:'Диаметр',
        itemParamOneValue: 15,
        itemParamOneUnit: 'см',
        itemParamTwoName: 'высота',
        itemParamTwoValue: 10,
        itemParamTwoUnit: 'см',
        itemQuantity: 1
    },
    {id:2,
        src: "./images/catalog/catalog-item-3.png",
        itemPrice: 1500,
        itemName:'Большая корзинка для игрушек',
        itemParamOneName:'Диаметр',
        itemParamOneValue: 30,
        itemParamOneUnit: 'см',
        itemParamTwoName: 'высота',
        itemParamTwoValue: 30,
        itemParamTwoUnit: 'см',
        itemQuantity: 1},

];

const initialState = {
    catalogItems: catalogItems,
    basketItems: basketItems,
};
const mockStore = configureStore();
const store = mockStore(initialState);

let results;
const renderHook = (hook) => {

    function HookWrapper() {
        results = hook();
        return null;
    }
    mount(
        <Provider store={store}>
            <HookWrapper/>
        </Provider>
    );
    return results;
};


renderHook(useCatalogHeadline)

const initialProps = {
     catalogItemsData:catalogItems,
     basketItemsData: catalogItems

};

jest.mock("react-redux", () => ({
    ...jest.requireActual("react-redux"),
    useSelector: jest.fn()
}));

const container = mount(
    <Provider store={store}>
        <CatalogHeadline {...initialProps}
        />
    </Provider>
);

describe('VideoBlock works', () => {

    it('should render VideoBlock Component', ()=> {
        const dom = container;
        const component = renderer.create(dom);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

    // it('VideoBlock props work', ()=> {
    //     expect(results.isPlaying).toBe(false)
    //     act(() => {
    //         results.handleMouseEnter();
    //     });
    //     expect(results.isPlaying).toBe(false)
    //     act(() => {
    //         results.handleMouseLeave();
    //     });
    //     expect(results.isPlaying).toBe(false)
    // });
});
