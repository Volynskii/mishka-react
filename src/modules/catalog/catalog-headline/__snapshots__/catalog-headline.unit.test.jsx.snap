// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoBlock works should render VideoBlock Component 1`] = `
<article
  className="catalog"
>
  <div
    className="catalog__headline"
  >
    Каталог товаров
    <h5>
      Каталог товаров
    </h5>
  </div>
  <ul
    className="catalog__list"
  >
    <li
      className="catalog__list__item"
    >
      <div
        className="catalog__list__item__image"
      >
        <img
          alt="картинка товара"
          className="catalog__list__item__image-picture"
          src="./images/catalog/catalog-item-1.png"
        />
      </div>
      <div
        className="catalog__list__item__wrapper"
      >
        <section
          className="catalog__list__item__param"
        >
          <div
            className="catalog__list__item__param__name"
          >
            Зайчик-попрыгайчик
          </div>
          <div
            className="catalog__list__item__param__description"
          >
            рост
             
            30
            см
            , 
            вес
             
            200
             
            г
          </div>
        </section>
        <ul
          className="catalog__list__item__order"
        >
          <li
            className="catalog__list__item__order__price"
          >
            1200
             р
          </li>
          <li
            className="catalog__list__item__order__basket tabs__item--active"
            onClick={[Function]}
          />
        </ul>
      </div>
    </li>
    <li
      className="catalog__list__item"
    >
      <div
        className="catalog__list__item__image"
      >
        <img
          alt="картинка товара"
          className="catalog__list__item__image-picture"
          src="./images/catalog/catalog-item-2.png"
        />
      </div>
      <div
        className="catalog__list__item__wrapper"
      >
        <section
          className="catalog__list__item__param"
        >
          <div
            className="catalog__list__item__param__name"
          >
            Корзинка для белья
          </div>
          <div
            className="catalog__list__item__param__description"
          >
            Диаметр
             
            15
            см
            , 
            высота
             
            10
             
            см
          </div>
        </section>
        <ul
          className="catalog__list__item__order"
        >
          <li
            className="catalog__list__item__order__price"
          >
            690
             р
          </li>
          <li
            className="catalog__list__item__order__basket tabs__item--active"
            onClick={[Function]}
          />
        </ul>
      </div>
    </li>
    <li
      className="catalog__list__item"
    >
      <div
        className="catalog__list__item__image"
      >
        <img
          alt="картинка товара"
          className="catalog__list__item__image-picture"
          src="./images/catalog/catalog-item-3.png"
        />
      </div>
      <div
        className="catalog__list__item__wrapper"
      >
        <section
          className="catalog__list__item__param"
        >
          <div
            className="catalog__list__item__param__name"
          >
            Большая корзинка для игрушек
          </div>
          <div
            className="catalog__list__item__param__description"
          >
            Диаметр
             
            30
            см
            , 
            высота
             
            30
             
            см
          </div>
        </section>
        <ul
          className="catalog__list__item__order"
        >
          <li
            className="catalog__list__item__order__price"
          >
            1500
             р
          </li>
          <li
            className="catalog__list__item__order__basket tabs__item--active"
            onClick={[Function]}
          />
        </ul>
      </div>
    </li>
  </ul>
</article>
`;
